internal/venice-client-common/src/main/java/com/linkedin/venice/httpclient5/HttpClient5Utils.java

```diff
@@ -40,9 +41,10 @@ public static class HttpClient5Builder {
      * We need to use a high connect timeout to avoid reconnect issue, which might result in confusing logging and unhealthy requests.
      * For now, we remove the functions updating to connect timeout to avoid mistakes.
      */
-    private final Timeout CONNECT_TIMEOUT_IN_MILLISECONDS = Timeout.ofMilliseconds(TimeUnit.MINUTES.toMillis(1)); // 1m
-                                                                                                                  // by
-                                                                                                                  // default
+    private final Timeout CONNECT_TIMEOUT_IN_MINUTES = Timeout.ofMinutes(1);
```

Comment:
@gaojieliu starting a comment here about java Doc for this class. How relevant is this currently and whether can this be removed?